styles/style.css

```diff
@@ -209,10 +296,40 @@ p {
 
 .facebook-link {
   color: var(--orange-dark);
+  text-align: right;
+}
+
+.fBLink:hover {
+  color: blueviolet;
 }
 
 /* Sidebar */
 
+.sideContainerOne {
+  display:flex;
```

Comment:
This won't need a `display: flex` because you are not using any flex property here.